### ISSUE TITLE
Prepare Changlelog for 1.8.0

### ISSUE THREAD

**Issue by Falke-Design** [Leaflet/Leaflet#8021]
## Breaking Changes
* Refactor `TapHold` handler to simulate `contextmenu` event on mobile Safari ([#7026](https://github.com/Leaflet/Leaflet/pull/7026) by [@johnd0e](https://github.com/johnd0e))
* Refactor `DivOverlay`/`Popup`/`Tooltip` ([#7540](https://github.com/Leaflet/Leaflet/pull/7540) by [@johnd0e](https://github.com/johnd0e))
	* Move `Popup` related options from `DivOverlay` to `Popup` ([#7778](https://github.com/Leaflet/Leaflet/pull/7778) by [@Falke-Design](https://github.com/Falke-Design))
	* Change `Tooltip` class from `leaflet-clickable` to `leaflet-interactive` ([#7719](https://github.com/Leaflet/Leaflet/pull/7719) by [@Falke-Design](https://github.com/Falke-Design))
	* `Map.closeTooltip` now requires a layer as argument ([#7533](https://github.com/Leaflet/Leaflet/pull/7533) by [@johnd0e](https://github.com/johnd0e))
* Improve error / argument handling for event listeners ([#7518](https://github.com/Leaflet/Leaflet/pull/7518) by [@johnd0e](https://github.com/johnd0e))
* Refactor `DomEvent.Pointer` which simulates `touchevents` on non-touch devices ([#7059](https://github.com/Leaflet/Leaflet/pull/7059), [#7084](https://github.com/Leaflet/Leaflet/pull/7084), [#7415](https://github.com/Leaflet/Leaflet/pull/7415) by [@johnd0e](https://github.com/johnd0e))
* Refactor `DomEvent.DoubleTap` which simulates `dblclick` event in not supported Browsers ([#7027](https://github.com/Leaflet/Leaflet/pull/7027) by [@johnd0e](https://github.com/johnd0e))
* Refactor `disableClickPropagation` ([#7439](https://github.com/Leaflet/Leaflet/pull/7439) by [@johnd0e](https://github.com/johnd0e))
* `Map.hasLayer()` and `LayerGroup.hasLayer()` now requires a layer as argument ([#6999](https://github.com/Leaflet/Leaflet/pull/6999) by [@johnd0e](https://github.com/johnd0e))
* `Class.include` keep options ([#7756](https://github.com/Leaflet/Leaflet/pull/7756) by [@johnd0e](https://github.com/johnd0e))
* Avoid modifying source props object in `Class.extend` ([#6766](https://github.com/Leaflet/Leaflet/pull/6766) by [@johnd0e](https://github.com/johnd0e))
* Correct `Browser.touch` ([#7029](https://github.com/Leaflet/Leaflet/pull/7029) by [@johnd0e](https://github.com/johnd0e))
* Get rid of android legacy hacks ([#7022](https://github.com/Leaflet/Leaflet/pull/7022) by [@johnd0e](https://github.com/johnd0e))

## API changes
* Make `DivOverlay` / `Tooltip` `interactive` ([#7531](https://github.com/Leaflet/Leaflet/pull/7531), [#7532](https://github.com/Leaflet/Leaflet/pull/7532) by [@johnd0e](https://github.com/johnd0e))
* Add `openOn`, `close`, `toggle` functions to `DivOverlay` ([#6639](https://github.com/Leaflet/Leaflet/pull/6639) by [@johnd0e](https://github.com/johnd0e))
* `DomEvent.off(el)` removes all listeners ([#7125](https://github.com/Leaflet/Leaflet/pull/7125) by [@johnd0e](https://github.com/johnd0e))
* Allow to prevent round-off errors by passing `false` to `Util.formatNum` / `toGeoJSON` ([#7100](https://github.com/Leaflet/Leaflet/pull/7100) by [@johnd0e](https://github.com/johnd0e))
* Add `autoPanOnFocus` to `Marker` ([#8042](https://github.com/Leaflet/Leaflet/pull/8042) by [@IvanSanchez](https://github.com/IvanSanchez))
* Add `referrerPolicy` to `TileLayer` ([#7945](https://github.com/Leaflet/Leaflet/pull/7945) by [@natevw](https://github.com/natevw))
* Add `playsInline` to `VideoOverlay` ([#7928](https://github.com/Leaflet/Leaflet/pull/7928) by [@Falke-Design](https://github.com/Falke-Design))
* Add `getCenter` to `ImageOverlay` ([#7848](https://github.com/Leaflet/Leaflet/pull/7848) by [@Falke-Design](https://github.com/Falke-Design))
* Fire a `tileabort` event when a `TileLayer` load is cancelled ([#6786](https://github.com/Leaflet/Leaflet/pull/6786) by [@dstndstn](https://github.com/dstndstn))
* Add `crossOrigin` to `Icon` ([#7298](https://github.com/Leaflet/Leaflet/pull/7298) by [@syedmuhammadabid](https://github.com/syedmuhammadabid))

## Improvements
* Avoid excessive `_update()` call in `GridLayer.onAdd` ([#7570](https://github.com/Leaflet/Leaflet/pull/7570) by [@johnd0e](https://github.com/johnd0e))
* Update `panInside` calculation ([#7469](https://github.com/Leaflet/Leaflet/pull/7469) by [@daverayment](https://github.com/daverayment))
* `TileLayer` allow template keys with spaces in URL ([#7216](https://github.com/Leaflet/Leaflet/pull/7216) by [@lubojr](https://github.com/lubojr))
* `Tooltips` can now bound to `ImageOverlays` because it can rely on parent layer `getBounds()` ([#7306](https://github.com/Leaflet/Leaflet/pull/7306) by [@IvanSanchez](https://github.com/IvanSanchez))
* Remove gap between Popup tip and content dialog ([#7920](https://github.com/Leaflet/Leaflet/pull/7920) by [@Malvoz](https://github.com/Malvoz))
* Fire `mousemove` through Canvas to map if it has no layers ([#7809](https://github.com/Leaflet/Leaflet/pull/7809) by [@johnd0e](https://github.com/johnd0e))
* Replace `div` with `span` in `Control.Layers` container to fix an HTML validation error ([#7914](https://github.com/Leaflet/Leaflet/pull/7914) by [@tmiaa](https://github.com/tmiaa))
* Improve memory footprint by removing `will-change` CSS property on tile images ([#7872](https://github.com/Leaflet/Leaflet/pull/7872) by [@janjaap](https://github.com/janjaap))
* Better icon image path detection ([#7092](https://github.com/Leaflet/Leaflet/pull/7092) by [@johnd0e](https://github.com/johnd0e))
* Makes `Browser` mutable ([#7335](https://github.com/Leaflet/Leaflet/pull/7335) by [@bozdoz](https://github.com/bozdoz))
* Add print styles to prevent printers from removing background-images in controls ([#7851](https://github.com/Leaflet/Leaflet/pull/7851) by [@Malvoz](https://github.com/Malvoz))
* Move attribution code from `Layer` to `Control.Attribution` ([#7764](https://github.com/Leaflet/Leaflet/pull/7764) by [@johnd0e](https://github.com/johnd0e))
* Minifiy marker icon svg ([#7600](https://github.com/Leaflet/Leaflet/pull/7600) by [@rala72](https://github.com/rala72)) 
* Refactor `vmlCreate()` so that it does not expose closure to `TypeError` ([#7279](https://github.com/Leaflet/Leaflet/pull/7279) by [@darcyparker](https://github.com/darcyparker))
* `Control.Layers` stop relying on Browser `android` and `touch` properties ([#7057](https://github.com/Leaflet/Leaflet/pull/7057) by [@johnd0e](https://github.com/johnd0e))
* `Tooltip` remove Browser `touch` checks ([#7535](https://github.com/Leaflet/Leaflet/pull/7535) by [@johnd0e](https://github.com/johnd0e))
* Converte `Plugins.md` to single files ([#7805](https://github.com/Leaflet/Leaflet/pull/7805) by [@Falke-Design](https://github.com/Falke-Design))
* Remove all references of Bower ([#7831](https://github.com/Leaflet/Leaflet/pull/7831) by [@jonkoops](https://github.com/jonkoops))

### Accessibility
* Add accessibility section to plugins guide ([#7277](https://github.com/Leaflet/Leaflet/pull/7277) by [@Malvoz](https://github.com/Malvoz))
* Auto pan to markers on focus by default for improved keyboard operability ([#8042](https://github.com/Leaflet/Leaflet/pull/8042) by [@IvanSanchez](https://github.com/IvanSanchez))
* Update `Marker` to default to `role="button"` & `alt="marker"` for an improved screen reader experience ([#7895](https://github.com/Leaflet/Leaflet/pull/7895) by [@tmiaa](https://github.com/tmiaa))
* Set `role="button"` for appropriate semantics on the `<a>` layers control ([#7850](https://github.com/Leaflet/Leaflet/pull/7850) by [@Malvoz](https://github.com/Malvoz))
* Generally enable outlines for keyboard users by not stripping `outline` on focus for keyboard events ([#7259](https://github.com/Leaflet/Leaflet/pull/7259) by [@jafin](https://github.com/jafin))
* Enable outlines on `leaflet-container` for keyboard users ([#7996](https://github.com/Leaflet/Leaflet/pull/7996) by [@Malvoz](https://github.com/Malvoz))
* Multiple enhancements to popup's close button ([#7794](https://github.com/Leaflet/Leaflet/pull/7794) by [@Falke-Design](https://github.com/Falke-Design))
* Use relative `font-size` units for resizable text ([#7800](https://github.com/Leaflet/Leaflet/pull/7800) by [@Chandu-4444](https://github.com/Chandu-4444))
* Apply `:hover` styles to `:focus` as well ([#7274](https://github.com/Leaflet/Leaflet/pull/7274) by [@Malvoz](https://github.com/Malvoz))
* Hide the decorative attribution separator from screen readers ([#7969](https://github.com/Leaflet/Leaflet/pull/7969) by [@Malvoz](https://github.com/Malvoz))
* Make the disabled state of zoom controls available to screen readers ([#7280](https://github.com/Leaflet/Leaflet/pull/7280) by [@akshataj96](https://github.com/akshataj96))
* Hide the +/- characters in zoom controls from screen readers to prevent erroneous announcements ([#7795](https://github.com/Leaflet/Leaflet/pull/7795) by [@Falke-Design](https://github.com/Falke-Design))

### Docs
* Upgrade Code of Conduct to Contributor Covenant v2 and improve its visibility ([#7984](https://github.com/Leaflet/Leaflet/pull/7984) by [@mourner](https://github.com/mourner))
* Lint examples ([#7827](https://github.com/Leaflet/Leaflet/pull/7827) by [@mourner](https://github.com/mourner))
* Update usability in Docs ([#7982](https://github.com/Leaflet/Leaflet/pull/7982),  [#7703](https://github.com/Leaflet/Leaflet/pull/7703), [#7950](https://github.com/Leaflet/Leaflet/pull/7950), [#7906](https://github.com/Leaflet/Leaflet/pull/7906), [#7907](https://github.com/Leaflet/Leaflet/pull/7907), [#7696](https://github.com/Leaflet/Leaflet/pull/7696), [#7816](https://github.com/Leaflet/Leaflet/pull/7816), [#7345](https://github.com/Leaflet/Leaflet/pull/7345), [#7815](https://github.com/Leaflet/Leaflet/pull/7815), [#7948](https://github.com/Leaflet/Leaflet/pull/7948), [#7901](https://github.com/Leaflet/Leaflet/pull/7901) by [@Falke-Design](https://github.com/Falke-Design), [@avioli](https://github.com/avioli), [@Malvoz](https://github.com/Malvoz), [@fulldecent](https://github.com/fulldecent), [@saerdnaer](https://github.com/saerdnaer), [@MxDui](https://github.com/MxDui))
* Typos / Fixes in Docs, Samples, ... ([#7263](https://github.com/Leaflet/Leaflet/pull/7263), [#7284](https://github.com/Leaflet/Leaflet/pull/7284), [#7339](https://github.com/Leaflet/Leaflet/pull/7339), [#7349](https://github.com/Leaflet/Leaflet/pull/7349), [#7381](https://github.com/Leaflet/Leaflet/pull/7381), [#7371](https://github.com/Leaflet/Leaflet/pull/7371), [#7485](https://github.com/Leaflet/Leaflet/pull/7485), [#7380](https://github.com/Leaflet/Leaflet/pull/7380), [#7578](https://github.com/Leaflet/Leaflet/pull/7578), [#7758](https://github.com/Leaflet/Leaflet/pull/7758), [#7602](https://github.com/Leaflet/Leaflet/pull/7602), [#7857](https://github.com/Leaflet/Leaflet/pull/7857), [#7860](https://github.com/Leaflet/Leaflet/pull/7860), [#7336](https://github.com/Leaflet/Leaflet/pull/7336), [#7819](https://github.com/Leaflet/Leaflet/pull/7819) by [@timgates42](https://github.com/timgates42), [@IvanSanchez](https://github.com/IvanSanchez), [@ipovos](https://github.com/ipovos), [@elfalem](https://github.com/elfalem), [@BakuCity](https://github.com/BakuCity), [@simon04](https://github.com/simon04), [@user073](https://github.com/user073), [@Dev-Steven](https://github.com/Dev-Steven), [@vanillajonathan](https://github.com/vanillajonathan), [@aquelle-cp](https://github.com/aquelle-cp), [@matkoniecz](https://github.com/matkoniecz), [@Falke-Design](https://github.com/Falke-Design))

#### Update API Docu:
*  Clarify `zoomend` event ([#7460](https://github.com/Leaflet/Leaflet/pull/7460) by [@xeruf](https://github.com/xeruf))
* Add `false` to `prefix` of `Control.Attribution` ([#7814](https://github.com/Leaflet/Leaflet/pull/7814) by [@Falke-Design](https://github.com/Falke-Design))
* `LayerGroup` inherit from `Interactive Layer` ([#7763](https://github.com/Leaflet/Leaflet/pull/7763) by [@johnd0e](https://github.com/johnd0e))
* Improve `Map.panInside` documentation ([#7397](https://github.com/Leaflet/Leaflet/pull/7397) by [@daverayment](https://github.com/daverayment))
* Update `Renderer` documentation to clarify `tolerance` option is for `Canvas` only ([#7515](https://github.com/Leaflet/Leaflet/pull/7515) by [@Hippl-Eric](https://github.com/Hippl-Eric))
* Add documentation for Event-Listener `propagate` argument ([#7103](https://github.com/Leaflet/Leaflet/pull/7103) by [@riffaud](https://github.com/riffaud))

## Bug fixes
* Fix `Marker` position while zooming ([#7967](https://github.com/Leaflet/Leaflet/pull/7967) by [@Falke-Design](https://github.com/Falke-Design))
* Fix `noConflict` ([#7855](https://github.com/Leaflet/Leaflet/pull/7855) by [@Falke-Design](https://github.com/Falke-Design))
* Fix opening / closing `Tooltip` while dragging map ([#7862](https://github.com/Leaflet/Leaflet/pull/7862) by [@Falke-Design](https://github.com/Falke-Design))
* Break the reference to the options of the Class prototype ([#7459](https://github.com/Leaflet/Leaflet/pull/7459) by [@Falke-Design](https://github.com/Falke-Design))
* Make `Tooltip` option `permanent` & `sticky`  working together ([#7563](https://github.com/Leaflet/Leaflet/pull/7563) by [@Falke-Design](https://github.com/Falke-Design))
* Check if map container is still connected with Leaflet in `Locate` event listener ([#7813](https://github.com/Leaflet/Leaflet/pull/7813) by [@Falke-Design](https://github.com/Falke-Design))
* `Tooltip.bindTooltip` should unbind existent tooltip ([#7633](https://github.com/Leaflet/Leaflet/pull/7633) by [@Falke-Design](https://github.com/Falke-Design))
* Correct `if` condition, to add zoom limits for Layer ([#7609](https://github.com/Leaflet/Leaflet/pull/7609) by [@vcoppe](https://github.com/vcoppe))
* `GridLayer` redraw tiles after changing `maxNativeZoom` ([#6443](https://github.com/Leaflet/Leaflet/pull/6443) by [@cherniavskii](https://github.com/cherniavskii))
* Fix `Popup` `keepInView` if the map needs to panned over a long distance ([#7792](https://github.com/Leaflet/Leaflet/pull/7792) by [@Falke-Design](https://github.com/Falke-Design))
* Tolerate wrong event names in  `add/removePointerListener` ([#7808](https://github.com/Leaflet/Leaflet/pull/7808) by [@johnd0e](https://github.com/johnd0e))
* Reset width & padding to prevent cascading CSS from breaking tile rendering ([#6843](https://github.com/Leaflet/Leaflet/pull/6843) by [@Spudley](https://github.com/Spudley))
*  Fix `mousedown` event calling after dragging `Canvas` map ([#7781](https://github.com/Leaflet/Leaflet/pull/7781) by [@johnd0e](https://github.com/johnd0e))
* Decrease `console.warn` pollution ([#7748](https://github.com/Leaflet/Leaflet/pull/7748) by [@johnd0e](https://github.com/johnd0e))
* Fix `contextmenu` event default-preventing when there are >1 target candidates ([#7544](https://github.com/Leaflet/Leaflet/pull/7544) by [@johnd0e](https://github.com/johnd0e))
* Prevent click on `Popup`-tip from firing on map. ([#7541](https://github.com/Leaflet/Leaflet/pull/7541) by [@johnd0e](https://github.com/johnd0e))
* Fix error by calling `Path.setStyle` before adding the layer to the map ([#6941](https://github.com/Leaflet/Leaflet/pull/6941) by [@NielsHolt](https://github.com/NielsHolt))
* Reset `BoxZoom` after cancel with ESC ([#7597](https://github.com/Leaflet/Leaflet/pull/7597) by [@Falke-Design](https://github.com/Falke-Design))

## Plugins
* Add / Update Plugins ([#7986](https://github.com/Leaflet/Leaflet/pull/7986), [#7956](https://github.com/Leaflet/Leaflet/pull/7956), [#7942](https://github.com/Leaflet/Leaflet/pull/7942), [#7888](https://github.com/Leaflet/Leaflet/pull/7888), [#7896](https://github.com/Leaflet/Leaflet/pull/7896), [#7885](https://github.com/Leaflet/Leaflet/pull/7885), [#7875](https://github.com/Leaflet/Leaflet/pull/7875), [#7874](https://github.com/Leaflet/Leaflet/pull/7874), [#7833](https://github.com/Leaflet/Leaflet/pull/7833), [#7834](https://github.com/Leaflet/Leaflet/pull/7834), [#7832](https://github.com/Leaflet/Leaflet/pull/7832), [#7812](https://github.com/Leaflet/Leaflet/pull/7812), [#7786](https://github.com/Leaflet/Leaflet/pull/7786), [#7802](https://github.com/Leaflet/Leaflet/pull/7802), [#6534](https://github.com/Leaflet/Leaflet/pull/6534), [#6547](https://github.com/Leaflet/Leaflet/pull/6547), [#7367](https://github.com/Leaflet/Leaflet/pull/7367), [#7392](https://github.com/Leaflet/Leaflet/pull/7392), [#7412](https://github.com/Leaflet/Leaflet/pull/7412), [#7631](https://github.com/Leaflet/Leaflet/pull/7631), [#7437](https://github.com/Leaflet/Leaflet/pull/7437), [#7408](https://github.com/Leaflet/Leaflet/pull/7408), [#6500](https://github.com/Leaflet/Leaflet/pull/6500), [#7957](https://github.com/Leaflet/Leaflet/pull/7957), [#7484](https://github.com/Leaflet/Leaflet/pull/7484), [#7236](https://github.com/Leaflet/Leaflet/pull/7236), [#7053](https://github.com/Leaflet/Leaflet/pull/7053), [#7050](https://github.com/Leaflet/Leaflet/pull/7050), [#7312](https://github.com/Leaflet/Leaflet/pull/7312), [#7623](https://github.com/Leaflet/Leaflet/pull/7623), [#7534](https://github.com/Leaflet/Leaflet/pull/7534), [#7626](https://github.com/Leaflet/Leaflet/pull/7626), [#7553](https://github.com/Leaflet/Leaflet/pull/7553), [#7423](https://github.com/Leaflet/Leaflet/pull/7423), [#7619](https://github.com/Leaflet/Leaflet/pull/7619), [#7065](https://github.com/Leaflet/Leaflet/pull/7065), [#7663](https://github.com/Leaflet/Leaflet/pull/7663), [#7690](https://github.com/Leaflet/Leaflet/pull/7690), [#7180](https://github.com/Leaflet/Leaflet/pull/7180), [#7714](https://github.com/Leaflet/Leaflet/pull/7714), [#7665](https://github.com/Leaflet/Leaflet/pull/7665), [#7179](https://github.com/Leaflet/Leaflet/pull/7179), [#6641](https://github.com/Leaflet/Leaflet/pull/6641), [#7647](https://github.com/Leaflet/Leaflet/pull/7647), [#7635](https://github.com/Leaflet/Leaflet/pull/7635), [#7473](https://github.com/Leaflet/Leaflet/pull/7473), [#7471](https://github.com/Leaflet/Leaflet/pull/7471), [#7363](https://github.com/Leaflet/Leaflet/pull/7363), [#7323](https://github.com/Leaflet/Leaflet/pull/7323), [#7295](https://github.com/Leaflet/Leaflet/pull/7295), [#7292](https://github.com/Leaflet/Leaflet/pull/7292), [#7291](https://github.com/Leaflet/Leaflet/pull/7291), [#7257](https://github.com/Leaflet/Leaflet/pull/7257), [#7276](https://github.com/Leaflet/Leaflet/pull/7276), [#7470](https://github.com/Leaflet/Leaflet/pull/7470), [#7700](https://github.com/Leaflet/Leaflet/pull/7700), [#7617](https://github.com/Leaflet/Leaflet/pull/7617))

## Workflow
* Add Bundlemon to watch bundle size ([#7934](https://github.com/Leaflet/Leaflet/pull/7934), [#7983](https://github.com/Leaflet/Leaflet/pull/7983), [#7905](https://github.com/Leaflet/Leaflet/pull/7905) by [@jonkoops](https://github.com/jonkoops))
* Add `npm run serve` to serve docs on localhost ([#7973](https://github.com/Leaflet/Leaflet/pull/7973) by [@Falke-Design](https://github.com/Falke-Design))
* Rename `master` branch to `main` ([#7921](https://github.com/Leaflet/Leaflet/pull/7921) by [@jonkoops](https://github.com/jonkoops))
* Upload files to AWS even if the file-size is the same ([#7853](https://github.com/Leaflet/Leaflet/pull/7853) by [@jonkoops](https://github.com/jonkoops))
* Remove `leaflet-include.js` from `debug` samples ([#7776](https://github.com/Leaflet/Leaflet/pull/7776) by [@Falke-Design](https://github.com/Falke-Design))
* Lint adjustments ([#7676](https://github.com/Leaflet/Leaflet/pull/7676), [#7743](https://github.com/Leaflet/Leaflet/pull/7743), [#7757](https://github.com/Leaflet/Leaflet/pull/7757) by [@jonkoops](https://github.com/jonkoops), [@mourner](https://github.com/mourner))
* Simplify release process ([#7711](https://github.com/Leaflet/Leaflet/pull/7711), [#7854](https://github.com/Leaflet/Leaflet/pull/7854), [#7727](https://github.com/Leaflet/Leaflet/pull/7727), [#8039](https://github.com/Leaflet/Leaflet/pull/8039) by [@mourner](https://github.com/mourner))
* Simplify docs update process on release ([#7730](https://github.com/Leaflet/Leaflet/pull/7730) by [@mourner](https://github.com/mourner))
* Update dependencies and add Dependabot config ([#7455](https://github.com/Leaflet/Leaflet/pull/7455), [#7653](https://github.com/Leaflet/Leaflet/pull/7653), [#7677](https://github.com/Leaflet/Leaflet/pull/7677), [#7725](https://github.com/Leaflet/Leaflet/pull/7725) by [@jonkoops](https://github.com/jonkoops))
* Split main workflow into multiple parallel jobs ([#7710](https://github.com/Leaflet/Leaflet/pull/7710) by [@jonkoops](https://github.com/jonkoops))
* Run CI on Github actions ([#7691](https://github.com/Leaflet/Leaflet/pull/7691), [#7654](https://github.com/Leaflet/Leaflet/pull/7654), [#7702](https://github.com/Leaflet/Leaflet/pull/7702) by [@jonkoops](https://github.com/jonkoops))
* Continue running tests even if one fails ([#7723](https://github.com/Leaflet/Leaflet/pull/7723) by [@jonkoops](https://github.com/jonkoops))
* Add https://github.com/Leaflet/Leaflet/labels/blocker check to release process ([#8019](https://github.com/Leaflet/Leaflet/pull/8019) by [@Malvoz](https://github.com/Malvoz))

## Tests
* simplify `.near` and `.nearLatLng` usage ([#7820](https://github.com/Leaflet/Leaflet/pull/7820) by [@johnd0e](https://github.com/johnd0e))
* Run tests on `Internet Explorer 11` ([#7741](https://github.com/Leaflet/Leaflet/pull/7741) by [@jonkoops](https://github.com/jonkoops))
* Run tests on `FirefoxNoTouch` ([#7736](https://github.com/Leaflet/Leaflet/pull/7736), [#7742](https://github.com/Leaflet/Leaflet/pull/7742) by [@johnd0e](https://github.com/johnd0e))
* Drop `PhantomJS` from test suite ([#7660](https://github.com/Leaflet/Leaflet/pull/7660), [#7724](https://github.com/Leaflet/Leaflet/pull/7724) by [@jonkoops](https://github.com/jonkoops))
* Enforce forbid-only rule in the continuous integration ([#7448](https://github.com/Leaflet/Leaflet/pull/7448) by [@johnd0e](https://github.com/johnd0e))
* Increase `captureTimeout` and `browserSocketTimeout` ([#7856](https://github.com/Leaflet/Leaflet/pull/7856) by [@Falke-Design](https://github.com/Falke-Design))
* Added / update tests ([#7790](https://github.com/Leaflet/Leaflet/pull/7790), [#7147](https://github.com/Leaflet/Leaflet/pull/7147), [#7721](https://github.com/Leaflet/Leaflet/pull/7721), [#7461](https://github.com/Leaflet/Leaflet/pull/7461), [#7126](https://github.com/Leaflet/Leaflet/pull/7126), [#7451](https://github.com/Leaflet/Leaflet/pull/7451), [#7450](https://github.com/Leaflet/Leaflet/pull/7450), [#7447](https://github.com/Leaflet/Leaflet/pull/7447), [#7438](https://github.com/Leaflet/Leaflet/pull/7438) by [@Falke-Design](https://github.com/Falke-Design), [@johnd0e](https://github.com/johnd0e), [@phloose](https://github.com/phloose))
* Added missing Test-Spec-Files to index.html ([#7845](https://github.com/Leaflet/Leaflet/pull/7845) by [@Falke-Design](https://github.com/Falke-Design))

**Comment by Falke-Design** [comment]
@mourner @Malvoz @IvanSanchez @jonkoops this are all commits since 1.7.1 I sorted them by category but now they need a deeper check and a better describtion, help is welcome. I suggest to simply edit the main post and add ✔️ to the commit.
Many commits are follow ups from one Feature / Bug or to revert some changes. So we need to check which commits are releated to the same topic and merge them together.

The biggest changes in this release are:
- [Refactor [touch/pointer] events](https://github.com/Leaflet/Leaflet/projects/1)
-  [Refactor DivOverlay/Popup/Tooltip](https://github.com/Leaflet/Leaflet/projects/2)
- Accessibility

Also I would like to add a thank you for @johnd0e to the Release notes. He refactored the touch events and DivOverlay without much help. Maybe I'm wrong but in my eyes kept @johnd0e Leaflet alive until we get a fresh wind in November.

**Comment by jonkoops** [comment]
Thanks @Falke-Design, much appreciated. I don't think I will have much time in the coming weeks, but I will try to see if I can walk through some of them. Unfortunately with the invasion of Ukraine I think @mourner has other priorities at the moment, so we'll have to get this done with the rest of the team.

**Comment by Falke-Design** [comment]
Yes I think too, but I still wanted to inform him even though he wont see it.

I already made the first points and it wasn't that much work for me. So if you don't have time, let me do it, is no problem ;)

**Comment by Malvoz** [comment]
Thanks for doing all this work @Falke-Design! I'm a bit busy atm but a couple of notes:

> - Accessibility changes

https://github.com/Leaflet/Leaflet/pull/7851 probably does not belong to that list, since it's just a small UX improvement for printed maps.

> Add fallbacks to font-size (https://github.com/Leaflet/Leaflet/pull/7800 by [@Chandu-4444](https://github.com/Chandu-4444))

I think the title of this item can be renamed "Use relative font-size units for resizable text".

**Comment by mourner** [comment]
Thanks so much for taking on this huge task @Falke-Design! I haven't read this through carefully, but several improvement tips I usually use when writing changelogs:

- Sort everything from most important to least important. For categories, it should roughly be 1) Breaking changes, 2) Features & Improvements, 3) Bugfixes, 4) Everything else. Within each category, also sort from most impactful to least.
- Where possible, reword changes to reflect _impact on the user_ rather than underlying change. E.g. not "Remove `will-change`", but rather "Improve memory footprint by removing `will-change` CSS property on tile images".
- Edit for consistency — e.g. same form (<Verb> ...), capitalized, etc.

**Comment by mourner** [comment]
Once the changelog is ready, I think we can simply tag a new version and do a release. Given the huge number of changes, we could first do a `v1.8.0-beta.1`, ask people to test it out, and then do a final a week later.

**Comment by mourner** [comment]
@Falke-Design let me know when you think the changelog is ready for publishing.

**Comment by mourner** [comment]
@IvanSanchez maybe you can have a quick editing run on it when you have a chance. We can create a "draft" release in the GitHub Releases section to quickly edit the changelog for convenience.

**Comment by Falke-Design** [comment]
I will finish it in the next days. My biggest problem will be _`reword changes to reflect impact on the user`_. I will reword them to show the impact on the user but I think someone has to go over it, to make the sentences grammitcal correct 😄

**Comment by Malvoz** [comment]
I've resolved https://github.com/Leaflet/Leaflet/issues/8021#issuecomment-1053606589, and https://github.com/Leaflet/Leaflet/issues/8021#issuecomment-1059765135:

> E.g. not "Remove will-change", but rather "Improve memory footprint by removing will-change CSS property on tile images"

and also turned the generic "Accessibility changes" list item with 10 or so items to individual list items with descriptions.

**Comment by Falke-Design** [comment]
Very nice! I want to change some small things in the next 2 hours and then we are ready 👍

**Comment by Falke-Design** [comment]
@mourner ready! 🎉 
Should we remove the *Plugins* section?

**Comment by Malvoz** [comment]
@Falke-Design I've made many small changes, to reword a few descriptions and to roughly "Sort everything from most important to least important" per https://github.com/Leaflet/Leaflet/issues/8021#issuecomment-1059765135, almost all of my edits are to the Accessibility section.

Also added some of the latest PRs that have been merged.

> Should we remove the _Plugins_ section?

I think so. It looks like there are only about 10 "plugin" entries in CHANGELOG.md and they're regarding breaking changes that affect plugins and otherwise compatibility, not regular additions/updates.
